code/ATMOSPHERICS/components/trinary_devices/filter.dm

```diff
@@ -43,7 +43,7 @@ Filter types:
 	update_icon()
 		if(stat & NOPOWER)
 			icon_state = "intact_off"
-		else if(node2 && node3 && node1)
+		else if(NODE_2 && NODE_3 && NODE_1)

```

Comment:
Excuse me ignorance, if these are all constants doesnt this mean this if statement is ALWAYS true?